src/Zygote.jl

```diff
@@ -24,8 +24,8 @@ include("lib/number.jl")
 include("lib/base.jl")
 include("lib/array.jl")
 include("lib/buffer.jl")
-include("lib/nnlib.jl")
 include("lib/broadcast.jl")
+include("lib/nnlib.jl")
```

Comment:
I move the `include("lib/nnlib.jl")` after `include("lib/broadcast.jl")` to make sure that `Numeric` visiable in `nnlib.jl`. But now the result of CI indicates that the `gradcheck` of `prod` failed.

Any idea?

Reply:
I have no idea why these things should be coupled, but... I don't know why the `prod` test ever passed, as its gradient is independent of `dims`. One attempt to fix it was here: https://github.com/FluxML/Zygote.jl/pull/112

Reply:
Interesting. If I comment out the newly added `gradtest` of `identity`, then all tests pass as they used to be. So my best guess is the randomness.

I'll set that test case as broken and leave if for #112 to fix it.